src/public/app/core/types/ScPlayer.ts

```diff
@@ -9,20 +9,20 @@ export interface ScPlayer {
 }
 
 export enum ScPlayerEvent {
-    StateChange = 'state-change',
-    Play = 'play',
-    PlayStart = 'play-start',
-    PlayResume = 'play-resume',
-    Pause = 'pause',
-    Finished = 'finish',
-    Seek = 'seek',
-    Seeked = 'seeked',
-    GeoBlocked = 'geo_blocked',
-    BufferingStart = 'buffering_start',
-    BufferingEnd = 'buffering_end',
-    AudioError = 'audio_error',
-    Time = 'time',
-    NoStreams = 'no_streams',
-    NoProtocol = 'no_protocol',
-    NoConnection = 'no_connection'
+    StateChange = <any>'state-change',
+    Play = <any>'play',
+    PlayStart = <any>'play-start',
+    PlayResume = <any>'play-resume',
+    Pause = <any>'pause',
```

Comment:
also let's avoid writing big definitions like this when we're only gonna use a few. We can add extras as we go, but it adds more difficulty in reading and refactoring if we have unused ones.